helix-term/src/commands.rs

```diff
@@ -3533,6 +3534,8 @@ fn goto_first_diag(cx: &mut Context) {
         None => return,
     };
     doc.set_selection(view.id, selection);
+    view.diagnostics_handler
+        .immidietly_show_diagnostic(doc, view.id);
```

Comment:
```suggestion
        .immediately_show_diagnostic(doc, view.id);
```

small typo in the member name that will need to be `<space>r`'d